code/modules/clothing/spacesuits/void/station_vr.dm

```diff
@@ -0,0 +1,42 @@
+//Exploration
+/obj/item/clothing/head/helmet/space/void/exploration
+	name = "exploration voidsuit helmet"
+	desc = "A radiation-resistant helmet made especially for exploring unknown planetary environments."
+	icon_state = "helm_explorer"
+	item_state = "helm_explorer"
+	armor = list(melee = 50, bullet = 15, laser = 25,energy = 35, bomb = 30, bio = 100, rad = 70)
+	light_overlay = "helmet_light_dual" //explorer_light
+
+/obj/item/clothing/suit/space/void/exploration
+	name = "exploration voidsuit"
+	desc = "A lightweight, radiation-resistant voidsuit, featuring the Explorer emblem on its chest plate. Designed for exploring unknown planetary environments."
+	icon_state = "void_explorer"
+	armor = list(melee = 50, bullet = 15, laser = 25,energy = 35, bomb = 30, bio = 100, rad = 70)
+	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank,/obj/item/device/suit_cooling_unit,/obj/item/stack/flag,/obj/item/device/healthanalyzer,/obj/item/device/gps,/obj/item/device/radio/beacon,/obj/item/weapon/shovel)
+
+/obj/item/clothing/suit/space/void/exploration/prepared
+	helmet = /obj/item/clothing/head/helmet/space/void/exploration
+	boots = /obj/item/clothing/shoes/magboots
+
+//Pilot
+/obj/item/clothing/head/helmet/space/void/pilot
+	desc = "An atmos resistant helmet for space and planet exploration."
+	name = "pilot voidsuit helmet"
+	icon_state = "rig0_pilot"
+	item_state = "pilot_helm"
+	armor = list(melee = 40, bullet = 5, laser = 20,energy = 5, bomb = 15, bio = 100, rad = 50)
+	max_heat_protection_temperature = FIRE_HELMET_MAX_HEAT_PROTECTION_TEMPERATURE
+	light_overlay = "helmet_light_dual"
+
+/obj/item/clothing/suit/space/void/pilot
+	desc = "An atmos resistant voidsuit for space and planet exploration."
+	icon_state = "rig-pilot"
+	item_state = "rig-pilot"
+	name = "pilot voidsuit"
+	armor = list(melee = 40, bullet = 5, laser = 20,energy = 5, bomb = 15, bio = 100, rad = 50)
```

Comment:
20 laser seems high. Even sec voidsuits only have 25. I suspect this was done because you have energy weapons and you're 'sacrificing' the low bullet value to say 'look it's not amazing armor' but since 99% of away missions are melee and energy, this armor is just pretty much exactly tailored to the threat.

Reply:
Fair enough.